configs/Gigabyte/GA-880GA-UD3H.conf

```diff
@@ -0,0 +1,116 @@
+# Configuration file for the Gigabyte GA-880GA-UD3H,
+# contributed by Clipo.
+# 2019-02-14 
+# Updates welcome.
+# 
+# Based on Gigabyte GA-870A-UD3.conf contributed by Gary Myers and Zeke Fast
+#
+#
+# CPU sensor
+
+chip "k10temp-*"
+
+   label temp1 "CPU Temp (K10)" 
+
+# Motherboard sensors
+
+chip "it8720-isa-*"
+
+### Voltages
+
+   label  in0  "Vcore"
+   label  in1  "DDR3 1.5V" # as labelled in the BIOS.
+   label  in2  "+3.3V"
+   label  in3  "+5V"     # Not shown in BIOS.
+   label  in4  "+12V"    # Not shown in BIOS
+   ignore in5		 # Ambiguous - ignoring.
+   ignore in6		 # Ambiguous - ignoring.
+   label  in7  "5VSB"    # Not shown in BIOS.
+   label  in8  "Vbat"    # Not shown in BIOS.
+   ignore cpu0_vid
+
+   # "Vcore", "DDR3 1.5V", "+3.3V" and "Vbat" are connected directly, so no compute
+   # line is needed for these. +5V and 5VSB are internal so we use the
+   # standard scaling factor. Scaling for +12V is apparently not standard
+   # this value was taken from the donor config file
```

Comment:
The scaling factor might be `8/2.7+1`, which we currently have in `GA-870A-USB3.conf`.